### ISSUE TITLE
How many computing resources did you use to train the model?

### ISSUE THREAD

**Issue by liuxin-a** [ShihaoZhaoZSH/Uni-ControlNet#10]
I would like to know if your model can be trained on Nvidia RTX 3090s?

**Comment by ShihaoZhaoZSH** [comment]
Certainly, it can be trained on NVIDIA RTX 3090s. The batch size have to be adjusted accordingly.

**Comment by songjiechong** [comment]
I would like to know the GPU numbers while training the model. I train a model with 2 Nvidia V100 and batch size = 1, but it still has the error "CUDA out of memory".

**Comment by ShihaoZhaoZSH** [comment]
> I would like to know the GPU numbers while training the model. I train a model with 2 Nvidia V100 and batch size = 1, but it still has the error "CUDA out of memory".

Please note that the batch size here is for a batch on a single GPU. A V100 with 24GB memory can handle the training with a batch size of 1.

**Comment by songjiechong** [comment]
> > I would like to know the GPU numbers while training the model. I train a model with 2 Nvidia V100 and batch size = 1, but it still has the error "CUDA out of memory".
> 
> Please note that the batch size here is for a batch on a single GPU. A V100 with 24GB memory can handle the training with a batch size of 1.

Thank you very much.

**Comment by liuxin-a** [comment]
> Certainly, it can be trained on NVIDIA RTX 3090s. The batch size have to be adjusted accordingly.
Thank you so much,I would like to know how many gpu hours it take you to train the model and which kind of GPU you used？Looking forward to your reply.
